datagrepper/templates/base.html

```diff
@@ -38,7 +38,7 @@
       <div class="container-narrow">
         <ul class="nav navbar-nav nav-justified nav-underline">
             <li class="nav-item"><a class="nav-link" href="{{ url_for('index') }}">Getting&nbsp;started</a></li>
-            <li class="nav-item active"><a class="nav-link" href="{{ url_for('raw') }}?rows_per_page=1&delta=127800">Feed</a></li>
+            <li class="nav-item active"><a class="nav-link" href="{{ url_for('raw') }}?rows_per_page=1&delta={{ default_query_delta }}">Feed</a></li>
```

Comment:
Shouldn't we just remove the `delta` argument?

Reply:
i have a feeling there was a reason for doing this -- but cant remember why now.